assets/section-footer.css

```
@@ -508,3 +527,24 @@
     }
   }
 }
+@media (min-width: 749px) {
+  .footer__content-bottom-wrapper--copyright {
+    justify-content: space-between;
+  }
+  .footer__socials-container {
+    display: flex;
+    justify-items: center;
+    align-items: center;
+    position: fixed;
+    right: 0;
+    top: 0;
+    bottom: 0;
+
+    background-color: white;
+    padding-right: 20%;
+    padding-left: 1rem;
+  }
+}
+.footer-padding {
+  padding-bottom: 0 !important;
```

```suggestion
  padding-bottom: 0;
```

Can we find another way to get this to work without having to use !important, for instance such as finding another parent class to target.